lib/modules/manager/nuget/extract.ts

```diff
@@ -123,14 +123,15 @@ export async function extractPackageFile(
   }
 
   let deps: PackageDependency[] = [];
-  let packageFileVersion = undefined;
+  let packageFileVersion;
```

Comment:
```suggestion
  let packageFileVersion: string | undefined;
```

I need to check if this value is obsolete and unused.

Reply:
Do you want me to use this in the meantime, or should I wait?